### ISSUE TITLE
Missing Purpose String in Info.plist File

### ISSUE THREAD

**Issue by 1284115201** [zoom/zoom-sdk-ios#38]
Missing Purpose String in Info.plist File - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSBluetoothPeripheralUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data will be required to include a purpose string.If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy).

This is the question I encountered when I submitted the audit and replied to me.

**Comment by 1284115201** [comment]
@chaobaizoom

**Comment by 1284115201** [comment]
@chaobaizoom I think the plist file in the given example is also accessing Bluetooth privileges, but no description is given.

**Comment by 1284115201** [comment]
NSCameraUsageDescription = "For people to see you during meetings, MobileRTC needs access to your camera.";

NSMicrophoneUsageDescription = "For people to hear you during meetings, MobileRTC needs access to your microphone.";

NSPhotoLibraryUsageDescription = "To share photos, MobileRTC needs access to your photo library.";

<key>NSBluetoothPeripheralUsageDescription</key>
	<string>bluetoothDecription</string>
	<key>NSCameraUsageDescription</key>
	<string>cameraDesciption</string>
	<key>NSMicrophoneUsageDescription</key>
	<string>microphoneDesciption</string>
	<key>NSPhotoLibraryUsageDescription</key>
	<string>photoLibraryDescription</string>

**Comment by chaobaizoom** [comment]
@1284115201  feel free to add description into your app info.plist.

**Comment by 1284115201** [comment]
> @1284115201 feel free to add description into your app info.plist.

I know I want to add, but I don't know where to use Bluetooth, and how to write a description.

**Comment by 1284115201** [comment]
> @1284115201 feel free to add description into your app info.plist.

SDK is exactly where Bluetooth is used.

**Comment by chaobaizoom** [comment]
yes, SDK need NSBluetoothPeripheralUsageDescription & NSCameraUsageDescription & NSMicrophoneUsageDescription & NSPhotoLibraryUsageDescription.  ：）
